frontend/src/components/elements/BokehChart/BokehChart.tsx

```diff
@@ -21,10 +21,13 @@ import { Map as ImmutableMap } from "immutable"
 
 interface Props {
   width: number
+  height: number | undefined
   element: ImmutableMap<string, any>
   index: number
 }
 
+const DEFAULT_HEIGHT = 400
```

Comment:
Does this mean that when in normal non-fullscreen mode the default height for a Bokeh chart will be 400px? If so, I think we shouldn't do that. Instead, just let Bokeh choose the height of the chart. This way Bokeh charts in Streamlit will work just like Bokeh charts anywhere else.

Reply:
Same with the other charts below.

Reply:
Ah I see. Yeah, I think removing the default makes sense. This way Bokeh decides what to use as default.

Same for other libraries in this PR.